misk-api/src/main/kotlin/misk/logging/Mdc.kt

```diff
@@ -0,0 +1,8 @@
+package misk.logging
+
+interface Mdc {
```

Comment:
do we need a method to clean?

Reply:
We will eventually I guess. Let me add it.

Reply:
Done